anaUltraScurve.py

```diff
@@ -249,9 +321,8 @@ def overlay_fit(VFAT, CHAN):
             noise[0] = param1
             fitENC.append(vToQm*param1*options.ztrim)
             pedestal[0] = param2
-            if noise[0] > 20.0 or ped_eff[0] > 50.0: mask[0] = True
```

Comment:
threshold - zTrim*noise is not the operational threshold and actually called the trimming point. The operational threshold is a value in charge units representing the 50% point on the scurve after set by the final threshold scan. Also, there is a point where the noise is so high, we want to mask that channel anyways.

The ped_eff cut is not exactly for "S-curve plots without a white band at the bottom". This cut catches a a couple specific scurve structures I have observed over the past year or so of studying hundreds of s-curve plots, from many different chambers, while developing these tools. We can discuss removing it at some point, but we need a good way to catch the I observed this to mask at some point. We also will need to make sure we mask channels which give no response during the scurve.